elements/rh-breadcrumb/docs/10-style.md

```diff
@@ -0,0 +1,96 @@
+## Style
+
+Breadcrumb navigation is composed of links, text, and caret icons as separators. There are two variants, default and subtle, which differ in text and icon color.
+
+### Anatomy
+
+<uxdot-example width-adjustment="295px">
+  <img src="{{ '../breadcrumbs-anatomy.svg'  | url }}" alt="Anatomy of breadcrumbs with numbered annotations pointing to various parts">
+</uxdot-example>
+
+1) Parent page
+2) Separator
+3) Current page
+   {.example-notes}
```

Comment:
```Suggestion
<figure>
  <uxdot-example width-adjustment="295px">
    <img src="{{ '../breadcrumbs-anatomy.svg'  | url }}" alt="Anatomy of breadcrumbs with numbered annotations pointing to various parts">
  </uxdot-example>
  <figcaption>
     <ol>
       <li>Parent page</li>
       <li>Seperator</li>
       <li>Current page</li>
     </ol>
  </figcaption>
</figure>
```

Reply:
Resolved in [5f8c425](https://github.com/RedHat-UX/red-hat-design-system/pull/1535/commits/5f8c425289faeec7e6f5986af26fb6ac49828142).